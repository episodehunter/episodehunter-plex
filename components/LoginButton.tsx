import * as React from 'react';

export function LogInButton(props: {buttonText: string, onClick: (e) => void}) {
  return (
    <button onClick={props.onClick}>{props.buttonText}</button>
  );
}
