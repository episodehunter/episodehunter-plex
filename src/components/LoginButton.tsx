import * as React from 'react';

export function LogInButton({onClick, buttonText}: {buttonText: string, onClick: (e) => void}) {
  return (
    <button onClick={onClick}>{buttonText}</button>
  );
}
