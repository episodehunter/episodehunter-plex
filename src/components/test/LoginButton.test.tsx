import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { LogInButton } from '../LoginButton';

test('that LogInButton renders', () => {
  const tree = renderer.create(
    <LogInButton buttonText="Button text!" onClick={() => {}} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
