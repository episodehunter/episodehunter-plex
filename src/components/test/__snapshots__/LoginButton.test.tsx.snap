// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`that LogInButton renders 1`] = `
<button
  onClick={[Function]}
>
  Button text!
</button>
`;
